src/lib/components/EventNetwork.svelte

```diff
@@ -2,8 +2,15 @@
   import { onMount } from "svelte";
   import * as d3 from "d3";
   import type { NDKEvent } from "@nostr-dev-kit/ndk";
-
+  import { nip19 } from "nostr-tools";
+  import { FeedType, standardRelays } from "$lib/consts";
+  import { levelsToRender } from "$lib/state";
+  let levels;
+  levelsToRender.subscribe((value) => {
+    levels = value;
+  });
```

Comment:
Svelte store values can be accessed more concisely, while preserving reactivity, like so:

```suggestion
  let levels = $levelsToRender;
```

In fact, we could simply use `$levelsToRender` directly in the component code, and do away with the `levels` middleman variable entirely.